### PR TITLE
Readme Rework

### PR DESCRIPTION
**Pull request by dnth**
This PR improves the readme page to be more visually appealing and also announces the availability of advanced features for free. 

I will re-work the notebooks in another PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -426,18 +405,60 @@ View the API docs [here](https://visual-layer.readme.io/docs/v1-api).
 
 ## Advanced Features
 
-The following are advanced functionalities of fastdup which are still in the beta testing phase.
-Sign up for free to be a beta tester and get early access. Drop us an email at info@visual-layer.com .
+Learn how to use fastdup as a poweruser!
+
+> **Note**: We have made the decision to offer our advanced features completely free of charge, without requiring a license key. Feel free to try them out and provide us with your valuable [feedback](https://visualdatabase.slack.com/join/shared_invite/zt-19jaydbjn-lNDEDkgvSI1QwbTXSY6dlA#/shared-invite/email)!
```
[the remark this answers is not in the document]
Comment (dnth) [reply README.md]: I edited the line saying we are releasing the features out from beta testing

[the remark this answers is not in the document]
Comment (dnth) [reply README.md]: I tested with 2D grid for the table and it looks like the following

image

[the remark this answers is not in the document]
Comment (dnth) [reply README.md]: In my opinion, it's a little harder to read due to the page width constraint on GitHub. On mobile, it's even harder to read. Do you think we should continue with the 2D grid?

[the remark this answers is not in the document]
Comment (dnth) [reply README.md]: Is this layout better @amiralush?

image

**Comment by dnth** [comment]
Is this still needed?

image

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amiralush) [reply README.md]: Hi, nope. I don't like this unfortunately. It's hard to read and not sure how much space it saves. 

Perhaps let's just half the notebooks icons size. So we could have something like this which is a bit more compact: 
Screenshot 2023-07-13 at 22 28 06

I guess that if you reduce the copy size of each example you could do same height (like in what I have attached) and two examples in a single row.

**Pull request merged**